package.json

```diff
@@ -7,11 +7,16 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "bootstrap": "^4.5.3",
+    "react": "^16.14.0",
```

Comment:
On these dependencies, which progress bar did you use in the end?

Reply:
"react-step-progress-bar": "^1.0.3"  this is used

Reply:
Super, we can tidy up any dependencies we don't need.